tests/pos/i13461-c.scala

```diff
@@ -0,0 +1,23 @@
+object Time:
```

Comment:
This test-case was made by me, based on i13461-b.scala. to show and test the behaviors of nested transparent calls.